test/index.spec.ts

```diff
@@ -0,0 +1,15 @@
+import * as semver from '../src';
+
+import { versions } from './versions';
+
+import { expect } from 'chai';
+
+describe('resin-semver', () => {
+	describe('.compare()', () => {
+		it('should not throw when provided with a version', () => {
+			versions.forEach((version) => {
+				expect(() => semver.compare(version, version)).to.not.throw();
```

Comment:
How about also adding `.and.to.equal(0)` or maybe this should be in a separate `it`?